allennlp/tests/modules/stacked_bidirectional_lstm_test.py

```diff
@@ -55,3 +57,43 @@ def test_stacked_bidirectional_lstm_can_complete_forward_pass_seq2vec(self):
         mask = torch.ones(4, 5)
         output = encoder(input_tensor, mask)
         assert output.detach().numpy().shape == (4, 18)
+
+    @flaky(max_runs=3)
+    @pytest.mark.parametrize("dropout_name", ('layer_dropout_probability',
+                                              'recurrent_dropout_probability'))
+    def test_stacked_bidirectional_lstm_dropout_version_is_different(self, dropout_name: str):
+        params = {'input_size': 10, 'hidden_size': 11, 'num_layers': 3}
```

Comment:
Unfortunately because we use `mypy` type checking, you cannot pass kwargs to constructors like this, because it makes mypy unable to check types. Could you pass them explicitly? This will fix the CI build.

Reply:
Just done a commit that will hopefully solve the mypy type checking.

Also thanks for reviewing my code.